Dockerfile

```diff
@@ -1,15 +1,11 @@
 FROM node:10-alpine
 
-RUN apk add --no-cache curl unzip
-
-WORKDIR /tmp
-RUN curl -L -o ./salien.zip https://github.com/meepen/salien-bot/archive/master.zip && \
-	unzip salien.zip && \
-	rm salien.zip
-
 WORKDIR /app
-RUN cp -R /tmp/salien-bot-master/* ./
+
+COPY package.json /app
 
 RUN yarn install
 
-ENTRYPOINT ["node", "headless.js"]
+COPY . /app
+
+CMD ["node", "headless.js"]
```

Comment:
IMHO in the case of using this Dockerfile with upcoming docker-compose file it's better to combine entrypoint and cmd in the next way
```
ENTRYPOINT ["node"]
CMD ["headless.js"]
```
When i start container with `docker-compose up` it would execute on up `node headless.js`
But if i wanna add params i run` docker-compose run server headless.js --some-param`
If we leave `CMD ["node", "headless.js"]`, to add param need to execute ` docker-compose run server node headless.js --some-param`

Reply:
@denas96 Yes, you have a point. I tought about this before creating PR but I've checked that Node official images use CMD: https://github.com/nodejs/docker-node/blob/65ba769f403f8c188d9be0b1ffb8a9cfc31bf703/10/alpine/Dockerfile#L69 so I used that approach too.